docs/visual-basic/programming-guide/language-features/data-types/tuples.md

```diff
@@ -41,6 +41,35 @@ Rather than using default names for a tuple's fields, you can instantiate a *nam
 
 [!code-vb[Instantiate](../../../../../samples/snippets/visualbasic/programming-guide/language-features/data-types/tuple1.vb#4)]
 
+## Inferred tuple field names
```

Comment:
The managed language team preferred the term "element" to "field" here. That's because the "field" names are still `Item1`, `Item2`, and so on from the `ValueTuple` struct. They aren't properties either, so we decided "elements" would be general enough and descriptive enough.

The 'field' term appears often.